coalahtml/_coalahtml/app/controllers/directives.js

```diff
@@ -18,10 +19,27 @@ angular.module('coalaHtmlApp')
           }
 
           element.html(prettyPrintOne(newValue, '', lineNums));
+
+          // Idea is to create elements dynamically and append to the code block.
+          var __className__ = $rootScope.THEME ? "highlight-line" : "highlight-line-dark";
```

Comment:
Haha alright. Later maybe. I pushed to show you the approach. :)
On 18 Jun 2016 00:28, "Attila" <notifications@github.com> wrote:

> In coalahtml/_coalahtml/app/controllers/directives.js
> <https://github.com/coala-analyzer/coala-html/pull/67#discussion_r67559307>
> :
>
> > @@ -18,10 +19,27 @@ angular.module('coalaHtmlApp')
> >            }
> >
> >            element.html(prettyPrintOne(newValue, '', lineNums));
> > +
> > +          // Idea is to create elements dynamically and append to the code block.
> > +          var __className__ = $rootScope.THEME ? "highlight-line" : "highlight-line-dark";
>
> why such a strange name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/67/files/e914e8579f2f39b65b3a897c384a03d4bb26d70f#r67559307>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHDgOU_MKiQYYY2YMKJzShOFPoZKxJLVks5qMu5VgaJpZM4I4nv0>
> .
>